36 create file for animation responsiveness

KristinShuyiHan opened the pull request:
Adding upward transition effect to frame3.html and frame3.js

dflores1229 committed:
added div for assets

dflores1229 committed:
update

dflores1229 committed:
added new frame (now frame3)

brianliu1412 committed:
Fixed background image not showing up

dflores1229 committed:
added all assets in frame1 to container

fangyuzhu1101 committed:
updates on css for cups size

dflores1229 committed:
adjusted background for mobile compat

dflores1229 committed:
implemented relativ position of start title/button

dflores1229 committed:
cleaned up code & assets positioning

dflores1229 committed:
rearranging assets

fangyuzhu1101 committed:
updates on the fixing frame3 layout

dflores1229 committed:
setup css files for each frame

dflores1229 committed:
added "boil" button

fangyuzhu1101 committed:
updates on frame 3 layout

fangyuzhu1101 committed:
updates on frame3.css

dflores1229 committed:
adjusted assets & styling frame5

dflores1229 committed:
adjusted frame1 assets and containers

dflores1229 committed:
began setting up container frame2

dflores1229 committed:
started setup for frame4 container

dflores1229 committed:
normed naming styles

ouyangca committed:
update js files for frame1 to add flickering effect

ouyangca committed:
create variables to store selected image

KristinShuyiHan committed:
Adding animation of teapot to frame 3

Adding upward transitional effect tp teapot and getting its original postion when clicking on pouring again in frame 3

KristinShuyiHan committed:
Adding animation of teapot to frame 3

Adding upward transitional effect tp teapot and getting its original postion when clicking on pouring again in frame 3

KristinShuyiHan committed:
Update frame3.html

KristinShuyiHan committed:
Adding pouring gif and leftwards animation to teapot in frame 3

Adding pouring gif and leftwards animation to teapot in frame 3

ouyangca committed:
merge with dev

CatFish47 commented:
Make sure to do `npm run lint` to see and change all linting errors! If you don't have ESLint installed yet, make sure to run `npm install` first.

ouyangca committed:
fix frame 2 merging issues with dev

ouyangca committed:
add boiling teapot

ouyangca committed:
add js for next link and edit frame 5

brianliu1412 committed:
CSS Changes: Styled frame 1 and 2. Added a font.

CatFish47 committed:
Merge branch 'dev' into 36-create-file-for-animation-responsiveness

CatFish47 committed:
eslinting autofix changes

CatFish47 committed:
Merge branch 'dev' into 36-create-file-for-animation-responsiveness

CatFish47 committed:
Add linting changes and fixes

CatFish47 committed:
Add css fixes

KristinShuyiHan committed:
pouring action initialized by clicking on teapot

KristinShuyiHan committed:
Merge branch '36-create-file-for-animation-responsiveness' of https://github.com/cse110-sp23-group18/cse110-sp23-group18 into 36-create-file-for-animation-responsiveness

CatFish47 committed:
Add documentation to the frame js

CatFish47 committed:
Cleans up code for frame1

KristinShuyiHan committed:
Delete teapotPouring.gif

KristinShuyiHan committed:
Add gif that only loop for once

Add gif that only loop for once

KristinShuyiHan committed:
Update frame3.html

CatFish47 committed:
Disable hover enlargement for now to focus mobile

CatFish47 committed:
fix lint from frame 3

CatFish47 committed:
Fixed overlay of tea reading

KristinShuyiHan committed:
Complete f3 's lint check & animation

Fix frame3 's linting issues and Complete animation of frame 3.

KristinShuyiHan committed:
Merge branch '36-create-file-for-animation-responsiveness' of https://github.com/cse110-sp23-group18/cse110-sp23-group18 into 36-create-file-for-animation-responsiveness

KristinShuyiHan committed:
Modifying f4.js to add interaction

modify the javaScript code  so that the teacup constantly enlarge and shrink alternately after the user clicks on it
